kinto/__main__.py

```diff
@@ -44,8 +45,14 @@ def main(args=None):
     parser_init.set_defaults(which='init')
 
     parser_migrate = subparsers.add_parser('migrate')
+    parser_migrate.add_argument('--dry-run',
+                                action='store_true',
+                                help='Information regarding version'
+                                     ' of the schema in user',
```

Comment:
Help would be better like `Simulate the migration operations and show information`